docs/configuring/README.md

```diff
@@ -202,6 +202,29 @@ And in YAML:
 
 **Note:** A globally-installed instance of ESLint can only use globally-installed ESLint plugins. A locally-installed ESLint can make sure of both locally- and globally- installed ESLint plugins.
 
+## Configuring Preprocessors
+
```

Comment:
Yes, I was thinking about doing it that way. I'll modify interface for plugin to have a single object for preprocessor that would have 3 properties: file extension, preprocessor, postprocessor.